x/nft/blockchain/codec.proto

```diff
@@ -12,14 +13,26 @@ message BlockchainToken {
 }
 
 message TokenDetails {
-    repeated Network networks = 1 [(gogoproto.nullable) = false];
+    Config config = 1 [(gogoproto.nullable) = false];
 }
 
-message Network { // these are all placeholders
-    bytes tickers = 1;
-    bytes bootstrap = 2;
-    bytes codec = 3;
-    bytes codecSettings = 4;
+enum Codec {
+    bns = 0;
+    lisk = 1;
+}
+
+enum Env {
+    mainnet = 0;
+    testnet = 1;
```

Comment:
q: how about string here too? there can be more than 1 testnet type.

Reply:
Same concern as above.

Reply:
Can this be done with a set of meaningful defaults? If we set some rules around the data allowed we can use string types safely. I would say we could go so far to say its only `production` and `nonproduction`. Any thing further is not really helpful to differentiate on.